edx_notifications/channels/tests/test_urban_airship.py

```diff
@@ -0,0 +1,48 @@
+"""
+Test cases for UA API
+"""
+from django.test import TestCase
+
+from edx_notifications import startup
+from edx_notifications.data import (
+    NotificationMessage,
+    NotificationType,
+)
+from edx_notifications.lib.publisher import (
+    register_notification_type,
+    publish_notification_to_tag)
+from testserver.views import CANNED_TEST_PAYLOAD
+
+TEST_TAG = 'cs101'
+
+
+class UrbanAirTestCases(TestCase):
+    """
+    Test cases for urban airship channel
+    """
+    def setUp(self):
+        """
+        Sets up test environments
+        :return:
+        """
+        startup.initialize(register_system_types=False)
+        self.msg_type = NotificationType(
+            name='open-edx.studio.announcements.new-announcement',
+            renderer='edx_notifications.openedx.course_announcements.NewCourseAnnouncementRenderer'
+        )
+        register_notification_type(self.msg_type)
+        self.msg = NotificationMessage(
+            namespace='foo/bar/baz',
+            msg_type=self.msg_type,
+            payload=CANNED_TEST_PAYLOAD['open-edx.studio.announcements.new-announcement']
+        )
+
+    def test_publish_notification_tag(self):
```

Comment:
we should add tests for other user cases like sending push notification to single user etc.